src/data/sql-quiz.ts

```diff
@@ -1053,6 +1053,39 @@ const sqlQuiz = [
     Explanation:
       "The SQL function SELECT COUNT(DISTINCT SUBJECT_ID) FROM table_name; is used to find the number of unique values in the SUBJECT_ID column, which represents the number of unique subjects taken by a student.",
     Link: "https://www.freecodecamp.org/news/sql-distinct-statement-how-to-query-select-and-count/"
+  },
+  {
+    Question:
+      "Which SQL clause is used to filter the result set of a SQL query by specifying a search condition?",
+    Answer: "WHERE",
+    Distractor1: "SELECT",
+    Distractor2: "FROM",
+    Distractor3: "GROUP BY",
+    Explanation:
+    "The WHERE clause is an essential part of SQL queries. It allows you to filter the rows in a result set by specifying a search condition that must be met for a row to be included in the result.",
+    Link: "https://www.freecodecamp.org/news/a-beginners-guide-to-sql/#chapter4crudoperationsinsql"
+  },
```

Comment:
It looks like we already have a similar question in the file.
Please remove this question 👍

Reply:
Made the specified changes.